container-storage-setup.sh

```diff
@@ -23,7 +23,7 @@ set -e
 
 # This section reads the config file $INPUTFILE
 # Read man page for a description of currently supported options:
-# 'man docker-storage-setup'
+# 'man container-storage-setup'
 
 POOL_LV_NAME="docker-pool"
```

Comment:
In fact this change probably should be the first one and should be done in a separate PR and then we can look at rest of the changes.